Makefile

```diff
@@ -243,7 +243,7 @@ clean:
 ## all: Run format codes, check codes, build Layotto codes for host platform with one command
 # ==============================================================================
 .PHONY: all
-all: format check.style check.unit check.lint build
+all: format check style.quickstart
```

Comment:
Do we need to run quickstart tests locally?

Reply:
Currently the quickstart test is essentially integration test. Running the integration test locally can help find CI issues as soon as possible.
Just personal opinion, waiting for others' feedback

Reply:
I understand but I remember that quick start checks will generate some files, we should handle these generated files gracefully

Reply:
Yes, I added them into the gitignore file. Maybe we can delete them before and after running the quickstart tests?

Reply:
Yes, you can try to delete them after testing finished or add them in make clean.

Reply:
yep, I think you are right. I added clean script. please review again